site/content/entry/yuzu-progress-report-mar-2020/index.md

```diff
@@ -0,0 +1,131 @@
+For example, one of our recent invalidations was needed to fix the reflections in Luigi’s Mansion 3, making certain parts of the game playable.
+
+Because dancing is important, [namkazt](https://github.com/namkazt) and [Rodrigo](https://github.com/ReinUsesLisp) have been fighting with the code, allowing us to enjoy 3D graphics in Hatsune Miku: Project DIVA MEGA39's, and as collateral, fixed some rendering issues in Bayonetta 1 and Super Smash Bros. Ultimate.
+
+{{< imgs
+    "./07.png| Miku Miku Dance!"
+  >}}
+
+{{< single-title-imgs
+    "Super Smash Bros. Ultimate's menu (Before & After)"
+    "./08.png"
+    "./09.png"
+  >}}
+
+[Kewlan](https://github.com/Kewlan), a regular in our Discord Server, submitted a small change to our input mapping GUI, changing the position of the shoulder buttons so they no longer make anyone angry.
+
+{{< imgs
+    "./11.png| Incorrect."
+    "./12.png| Much better."
+  >}}
+
+We also have a cool improvement by [Tobi](https://github.com/FearlessTobi).
+The Switch is a surround sound compatible console and our implementation of downmixing removed any channel that wasn’t left or right.
+By keeping the extra channels and mixing them in the left and right stereo outputs, the sound in tons of games, including Luigi’s Mansion 3, Sonic Forces, and Bayonetta, has been improved.
+
+## A peculiar case of "Even a broken clock is right twice a day!"
+
+The Nintendo Switch supports `ASTC`, a type of texture format generally used in mobile hardware such as phones and tablets in order to reduce bandwidth usage (a limited resource in those devices).
+Since it's a heavily compressed format, decoding it via the CPU, like it is normally done on PC, is a very slow task.
+
+To give an example, Astral Chain is a game that uses `ASTC` very extensively and in very strange ways, as previous fixes and optimizations have shown.
+On NVIDIA and AMD graphics hardware, the first scene of a new start of the game normally takes up to `17` seconds to load, as the GPU has to wait for the CPU to decode all the `ASTC` textures first, slowing down the whole process.
+
+Thanks to [Rodrigo's](https://github.com/ReinUsesLisp) research, it was discovered that on any recent Intel GPU this same operation is immediate and occurs with no delay, thanks to it being the only vendor providing a native `ASTC` decoder in their hardware.
+That’s right, the best way to experience Astral Chain in yuzu is with a small, and typically inferior, iGPU (like the ones that come in any desktop or notebook CPU from Intel).
+
+On the topic of formats, implementing the `RG32` and `RGB32` vertex formats fixed a room that refused to load in The Legend of Zelda: Link’s Awakening in Vulkan.
+Great work [Rodrigo](https://github.com/ReinUsesLisp)!
+
+{{< imgs
+    "./13.png| Works!"
+  >}}
+  
+## Paving the road for even bigger changes
+
+[Rodrigo](https://github.com/ReinUsesLisp)'s initial work on Transform Feedback is finished and it's partially working. 
```

Comment:
```suggestion
[Rodrigo](https://github.com/ReinUsesLisp)'s initial work on Transform Feedback has concluded and partially working. 
```

Reply:
I like it, but it misses and *is* partially working.